src/services/link/index.js

```diff
@@ -112,7 +115,7 @@ module.exports = function setup() {
          *  }
          */
 		create(data, params) {
-			return new Promise(async (resolve, reject) => {
+			return new Promise(async (resolve) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Promise executor functions should not be async.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5361392)